src/Microsoft.AspNetCore.Mvc.Core/Internal/FileResultExecutorBase.cs

```diff
@@ -17,6 +17,8 @@ namespace Microsoft.AspNetCore.Mvc.Internal
 {
     public class FileResultExecutorBase
     {
+        internal const string ProcessRangeRequestsSwitch = "Switch.Microsoft.AspNetCore.Mvc.TurnOnRangeProcessing";
```

Comment:
Is this name confusing? The switch only applies to `FileContentResult` and `FileStreamResult`. 

cc @Eilon

Reply:
Yeah that's fine actually. Or maybe instead of `TurnOn` use `Enable`?

Reply:
Changing it to `EnableRangeProcessing`